src/Testcontainers.Seq/SeqBuilder.cs

```diff
@@ -0,0 +1,72 @@
+namespace Testcontainers.Seq;
+
+/// <inheritdoc cref="ContainerBuilder{TBuilderEntity, TContainerEntity, TConfigurationEntity}" />
+[PublicAPI]
+public sealed class SeqBuilder : ContainerBuilder<SeqBuilder, SeqContainer, SeqConfiguration>
+{
+    public const string SeqImage = "datalust/seq:2024.2.11456";
```

Comment:
You should use the latest image
datalust/seq:latest

Reply:
Unfortunately, this is incorrect. We pin the image version (if tags/versions are available), as mentioned in our [best practices](https://dotnet.testcontainers.org/api/best_practices/). We avoid using `latest` whenever possible.

Reply:
Ah sorry my bad